### PR TITLE
#159951824 Add user commenting on article feature.

### PR DESCRIPTION
**Pull request by abulojoshua1**
### What does this PR do?
- Adds commenting functionality to Authors Haven

### Description of Task to be completed?
- Enable a logged in user comment on articles

### How should this be manually tested?
-  Navigate to an article (read an article)
- scroll to the bottom and type a comment in the input field provided

### Any background context you want to provide?
- I created a commenting component
### What are the relevant pivotal tracker stories?
- [#159951824](https://www.pivotaltracker.com/story/show/159951824)
screen shot 2018-09-13 at 12 05 51 pm

### CODE REVIEW

**Comment by RutaleIvanPaul** [comment]
@abulojoshua1 , we need the Screenshot of coverage. eof lines are missing

**Comment by abulojoshua1** [comment]
Ok, Right away on that screenshot!

File: src/CSS/CreateComment.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+.loadMore:hover {
+    color: #717477;
+}
+.CommentersName:hover {
+    color: #00bcd4;
+}
+.errors {
+    color: red;
+}
```
Comment (RutaleIvanPaul) [inline src/CSS/CreateComment.css:96]: eof misssing here

Comment (abulojoshua1) [reply src/CSS/CreateComment.css:96]: That file already has an EOF on it, please check it again @RutaleIvanPaul.

Comment (abulojoshua1) [reply src/CSS/CreateComment.css:96]: All my files have EOF lines on them, please check and tell me which one in particular!

**Comment by AGMETEOR** [comment]
@abulojoshua1 looks great. Can we remove the <user> "said" and just leave <user> and the comment...

**Comment by abulojoshua1** [comment]
I've just done that now.

**Review by AGMETEOR — APPROVED** [review]

**Review by lytes20 — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]

**Review by ssewilliam — APPROVED** [review]

**Comment by AGMETEOR** [comment]
@abulojoshua1 I think the comments are not being ordered well based on the time they were made. Please check that.

**Comment by abulojoshua1** [comment]
Check it out well @AGMETEOR, the once the components load, they are ordered based on time

**Pull request merged**